doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1724,9 +1724,53 @@ Ltac2 features a toplevel loop that can be used to evaluate expressions.
    together with its type. This command is pure in the sense that it does not
    modify the state of the proof, and in particular all side-effects are discarded.
 
+Compilation
+-----------
+
+Ltac2 definitions can be compiled to OCaml, the compiled definitions
+will then run natively instead of using the interpreter.
+
+.. cmd:: Ltac2 Compile {* @qualid }
+
+   Compile the given Ltac2 definitions. Compilation does not survive
+   state resets (end of file, end of module, end of section).
+
+   Ltac2 code in an :ref:`antiquotation <term-antiquotations>` will
+   not be compiled. It will also not be seen by dependency analysis,
+   so Ltac2 definitions appearing in an antiquotation which appears in
+   an argument of `Ltac2 Compile` will not be automatically compiled.
```

Comment:
```suggestion
   not be compiled. Nor will it be detected by dependency analysis,
   so Ltac2 definitions in antiquotations in tactics passed to
   `Ltac2 Compile` will not be automatically compiled.  In this case,
   the compile command should explicitly list such definitions.
```

Will the compiled version give correct results (though perhaps slower) if there is such a definition?  How about if ltac2 calls ltac1?

Reply:
The compiled version should always give correct results except for the explicitly documented issues (backtraces etc)